packages/display/test/DisplayObject.tests.ts

```diff
@@ -247,23 +247,29 @@ describe('DisplayObject', function ()
 
     describe('destroy', function ()
     {
-        it('should trigger destroyed listeners', function ()
+        it('should trigger destroyed listeners once destruction is complete', function ()
```

Comment:
I'm happy to make a second test instead, but I found there to be quite a bit of overlap so I extended this one instead

Reply:
Would it be possible to do a different test instead of modifying this one? The original test is simply testing if destroyed got called, and your test is check if the destroyed flag is true at the time of the callback. Those feel like different cases.

Reply:
@bigtimebuddy updated, thanks for taking a look!